### DISCUSSION TITLE
GitHub Classroom Setup Help (Free version)

### DISCUSSION THREAD

**Discussion by JulDon** [community/community#70508] [GitHub Education]
### Hi everyone,

Hello. I plan to set up the free version of GitHub Classroom for my 14 students that are taking HTML/JavaScript. Unfortunately, my school doesn't have a license. I want to have a place where each student can store their work and I can go in and look at their code and give them feedback. Want to make sure I am setting it up correctly. Can anyone please help or do you know of any good training videos on this? I'm hoping to have something set up by this weekend. Thanks so much for any advice or help you can give me!

**Comment by robskaar** [comment]
Hi, there's surely a lot of youtube videos, but nothing beats documentation, hope you have or will figure it out:
https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/manage-classrooms
